qa/tasks/cephfs/test_killpoints_dir_export.py

```diff
@@ -0,0 +1,136 @@
+
+"""
+Test to exercise killpoints available during directory export code path
+After hitting a killpoint, MDS crashes and cluster must find a standby MDS
+to become active. Exported data must be available after mds replacement.
+
+"""
+import logging
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+
+log = logging.getLogger(__name__)
+
+
+class TestKillPoints(CephFSTestCase):
+    # Two active MDS and a standby
+    MDSS_REQUIRED = 3
+
+    init = False
+
+    def setup_cluster(self):
+        # Set Multi-MDS cluster
+        self.fs.set_allow_multimds()
```

Comment:
@gregsfortytwo thanks for the suggestion. I am making changes for import killpoints and configurable number of files.
@jcsp I will shorten the test name, e.g. test_kllpoints.py? I will add a reference for teuthology. Also, will add generic helper for fs flags.